sandboxed_containers/upgrade-sandboxed-containers.adoc

```diff
@@ -0,0 +1,25 @@
+[id="upgrade-sandboxed-containers"]
+= Upgrade {sandboxed-containers-first}
+include::modules/common-attributes.adoc[]
+:context: upgrade-sandboxed-containers
+
+toc::[]
+
```

Comment:
You need a sentence to introduce the assembly. (It's what shows up in Google searches.)